pkg/cmd/cli/describe/describer.go

```diff
@@ -363,17 +369,66 @@ func (d *ImageDescriber) Describe(namespace, name string) (string, error) {
 		return "", err
 	}
 
-	return describeImage(image)
+	return describeImage(image, "")
 }
 
-func describeImage(image *imageapi.Image) (string, error) {
+func describeImage(image *imageapi.Image, imageName string) (string, error) {
 	return tabbedString(func(out *tabwriter.Writer) error {
 		formatMeta(out, image.ObjectMeta)
 		formatString(out, "Docker Image", image.DockerImageReference)
+		if len(imageName) > 0 {
+			formatString(out, "Image Name", imageName)
+		}
+		formatString(out, "Parent Image", image.DockerImageMetadata.Parent)
+		formatString(out, "Layer Size", units.HumanSize(float64(image.DockerImageMetadata.Size)))
```

Comment:
Can you do a pull for that and then I'll come back and change this around?

----- Original Message -----
> >  	return tabbedString(func(out *tabwriter.Writer) error {
> >  		formatMeta(out, image.ObjectMeta)
> >  		formatString(out, "Docker Image", image.DockerImageReference)
> > +		if len(imageName) > 0 {
> > +			formatString(out, "Image Name", imageName)
> > +		}
> > +		formatString(out, "Parent Image", image.DockerImageMetadata.Parent)
> > +		formatString(out, "Layer Size",
> > units.HumanSize(float64(image.DockerImageMetadata.Size)))
> 
> image.DockerImageMetadata.VirtualSize, which... we don't have in our struct
> :-1:
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/2356/files#r30630943